### ISSUE TITLE
Erro CHARSET new server AWS Lightsail

### ISSUE THREAD

**Issue by gbrn** [WWBN/AVideo#7321]
hi @DanielnetoDotCom ,

I'm doing some tests with the change of the AVIDEO platform from the Locaweb server to the AWS Lightsail server.
Checked some Latin characters display errors: `episÃƒÂ³dios`
I believe this problem must be due to some specific configuration on the new server.

I've already tried some settings in apache and PHP, but I can't solve it. I've verified that the default encoding of AVIDEO is: `<meta charset="utf-8">`

I already tried the settings in apache:
`AddDefaultCharset ISO-8859-1`
`php_admin_value default_charset ISO-8859-1`

Can you tell me if there is any other location in UBUNTU 20.04 LTS that the configuration should be changed?

**Comment by DanielnetoDotCom** [comment]
did you import the database form somewhere else?

usually, this happens when you use different MySQL versions.

as a workaround try to add this on your `videos/configuration.php` file


`$global['mysqli_charset'] = 'latin1';`

**Comment by gbrn** [comment]
Adding the line in configuration.php solved the problem.

I imported the database from the old server. The platform files are more current, I updated through PULL.
MySql used previously was 5.0 and now 8.0. Is it because of the version?
Something I have to do to fix, reverse this problem? I want to run the system now in the most correct way.

**Comment by DanielnetoDotCom** [comment]
Just keep the $global['mysqli_charset'] = 'latin1'; and you will be fine.

**Comment by gbrn** [comment]
@DanielnetoDotCom I'm having a problem with accented Latin characters again.
On moving to the new server, this problem came up again.

I added the line `$global['mysqli_charset'] = 'latin1';` in configuration.php.
Fixed the texts on the website, but the curious thing is that only the left sidebar kept the wrong characters in categories:
![image](https://user-images.githubusercontent.com/37880073/216802311-84b3b85c-7e53-41ce-a334-56c488b91f87.png)
![image](https://user-images.githubusercontent.com/37880073/216802335-81c67bc0-21a1-4075-ad89-30a101f171b7.png)
![image](https://user-images.githubusercontent.com/37880073/216802424-ae985680-1d59-4fbb-9d32-958bab394923.png)

Is this something in the code or some setting?
